packages/protocol/test/resources/layout/contracts_appended/Migrations.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.5.3;
```

Comment:
Why is this contract needed?

Reply:
Ah, thought maybe it was necessary for `truffle compile` but doesn't appear to be so.